src/routes/index.js

```diff
@@ -50,9 +51,11 @@ export function configureRoutes(app: Koa) {
 function configureTechnicalRoutes(app: Koa) {
   const dockerFlow = dockerFlowRoutes();
   const cspReport = cspReportRoutes();
+  const profile = profileRoutes();
 
   app.use(dockerFlow.routes()).use(dockerFlow.allowedMethods());
   app.use(cspReport.routes()).use(cspReport.allowedMethods());
+  app.use(profile.routes()).use(profile.allowedMethods());
```

Comment:
This probably has to go to `configureUserFacingRoutes` below, otherwise it won't have CORS nor versioning.

(should I have added comments to these functions ? ;-) )